katdal/datasources.py

```diff
@@ -643,6 +644,16 @@ def from_url(cls, url, chunk_store='auto', upgrade_flags=True, **kwargs):
                 telstate = katsdptelstate.TelescopeState(url_parts.netloc, db)
             except katsdptelstate.ConnectionError as e:
                 raise DataSourceNotFound(str(e))
+        elif url_parts.scheme in {'http', 'https'}:
+            # Open endpoint as an S3 object store (with auth info in kwargs)
+            endpoint = urllib.parse.urlunparse(url_parts[:2] + 4 * ('',))
+            rdb_store = S3ChunkStore.from_url(endpoint, **kwargs)
+            rdb_path = urllib.parse.urlunparse(url_parts[:3] + 3 * ('',))
+            telstate = katsdptelstate.TelescopeState()
+            with rdb_store.request('RDB', 'GET', rdb_path) as response:
```

Comment:
Should a failure get turned into `DataSourceNotFound`?

Reply:
I like that.